apps/betterangels-backend/accounts/tests/test_models.py

```diff
@@ -23,33 +24,46 @@ def test_full_name(self) -> None:
         user_2 = baker.make(User, first_name="Dale", middle_name="Bartholomew", last_name="Cooper")
         self.assertEqual(user_2.full_name, "Dale Bartholomew Cooper")
 
-    def test_is_outreach_authorized(self) -> None:
+    @parametrize(
+        "user_is_in_authorized_org, user_is_in_unauthorized_org, user_has_accepted_tos, user_has_accepted_privacy_policy, should_succeed",
+        [
+            (True, True, True, True, True),
+            (True, False, True, True, True),
+            (False, True, True, True, False),
+            (True, True, False, True, False),
+            (True, True, True, False, False),
+        ],
+    )
+    def test_is_outreach_authorized(
+        self,
+        user_is_in_authorized_org: bool,
+        user_is_in_unauthorized_org: bool,
+        user_has_accepted_tos: bool,
+        user_has_accepted_privacy_policy: bool,
+        should_succeed: bool,
+    ) -> None:
         authorized_org = organization_recipe.make(name="authorized org")
         unauthorized_org = organization_recipe.make(name="unauthorized org")
 
-        (
-            user_in_auth_org,
-            user_in_unauth_org,
-            user_in_both_orgs,
-            user_in_no_orgs,
-        ) = baker.make(User, _quantity=4)
+        user = baker.make(
+            User,
+            has_accepted_tos=user_has_accepted_tos,
+            has_accepted_privacy_policy=user_has_accepted_privacy_policy,
+        )
 
-        authorized_org.add_user(user_in_auth_org)
-        authorized_org.add_user(user_in_both_orgs)
-        unauthorized_org.add_user(user_in_both_orgs)
-        unauthorized_org.add_user(user_in_unauth_org)
+        if user_is_in_authorized_org:
+            authorized_org.add_user(user)
 
-        remove_organization_permission_group(unauthorized_org)
+        if user_is_in_unauthorized_org:
+            unauthorized_org.add_user(user)
 
         permission_group_recipe.make(
             name="unauthorized permission group",
             organization=unauthorized_org,
         )
+        remove_organization_permission_group(unauthorized_org)
```

Comment:
a little confused how this test was working before, is this the same as not creating the permission group on line 60?

Reply:
old test didn't accurately represent the property functionality. Fixed property and updated test. Thanks for drawing attention to this 🙏